sdk/nodejs/iot/zMixins.ts

```diff
@@ -0,0 +1,68 @@
+import * as pulumi from "@pulumi/pulumi";
```

Comment:
in general, we require a copyright notification on every file.  though if we've been inconsistent, then this will be on me to fixup.

Reply:
How should the copyright header look like? I found various examples-  ranging from zero copyright header, to a one-liner, to a whole paragraph.

Reply:
Should be this one:

```
// Copyright 2016-2018, Pulumi Corporation.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Thanks!

Reply:
Fixed in 8e51bc3ea8e7db040ee3e672ebe94b9b089751c1

Reply:
BTW - should I resolve conversations that I fixed or will you?

Reply:
If it's trivial, you're welcome to.  if it seems like more of something that should have someone check on, we can do it :)